src/lib/utils/mem_ops.cpp

```diff
@@ -58,9 +58,13 @@ uint8_t ct_compare_u8(const uint8_t x[],
                       size_t len)
    {
    volatile uint8_t difference = 0;
```

Comment:
We should support using `timingsafe_bcmp` here (and any other similar APIs that may exist, though I don't know of any)

Reply:
Actually `timingsafe_bcmp` turns out to be ... well not unsafe exactly but certainly very fragile wrt future compiler optimizations, so we probably don't want to do this.